notesapi/v1/models.py

```diff
@@ -11,12 +12,16 @@ class Note(models.Model):
     course_id = models.CharField(max_length=255, db_index=True)
     usage_id = models.CharField(max_length=255, help_text="ID of XBlock where the text comes from")
     quote = models.TextField(default="")
+    parent = models.ForeignKey('Note', blank=True, null=True, help_text="Parent note, if this is a comment")
```

Comment:
Dude. Don't review the code. It's not ready to be merged at all. Jamie
asked to see it. It's work in progress.

On Thursday, October 22, 2015, Shawn Milochik <notifications@github.com>
wrote:

> In notesapi/v1/models.py
> <https://github.com/mitodl/edx-notes-api/pull/1#discussion_r42820361>:
>
> > @@ -11,12 +12,16 @@ class Note(models.Model):
> >      course_id = models.CharField(max_length=255, db_index=True)
> >      usage_id = models.CharField(max_length=255, help_text="ID of XBlock where the text comes from")
> >      quote = models.TextField(default="")
> > +    parent = models.ForeignKey('Note', blank=True, null=True, help_text="Parent note, if this is a comment")
>
> Long line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mitodl/edx-notes-api/pull/1/files#r42820361>.
>


-- 
Sent via mobile phone.